pandas/tests/indexes/datetimes/test_timezones.py

```diff
@@ -706,6 +706,32 @@ def test_join_utc_convert(self, join_type):
         assert isinstance(result, DatetimeIndex)
         assert result.tz.zone == 'UTC'
 
+    @pytest.mark.parametrize("dtype", [
+        None, 'datetime64[ns, CET]',
+        'datetime64[ns, EST]', 'datetime64[ns, UTC]'
+    ])
+    def test_date_accessor(self, dtype):
+        # Regression test for GH#21230
+        expected = np.array([date(2018, 6, 4), pd.NaT], ndmin=1)
```

Comment:
is the `ndmin=1` needed here?

Reply:
I needed it to avoid a shape of (1,) when creating a numpy array with only 1 element. I'll remove it now.